tc3117.md

```diff
@@ -0,0 +1,23 @@
+# Open Source Development
+
+Name: Tattie Chitrakorn
+
+UNI: tc3117
+
+GitHub: [link](https://github.com/tchitrakorn)
+
+GitHub README: [link](https://github.com/tchitrakorn/tchitrakorn/blob/main/README.md)
+
+
+## Project Proposal
+- Idea 1: [ITZMA Lint](https://github.com/tchitrakorn/project-proposals-s2023/blob/hw1/projects/python/itzma-lint.md)
```

Comment:
use relative links like `./projects/python/itzma-lint.md` otherwise it will link to your fork of this repository instead of this repository directly